src/main/java/pinball/BodyType.java

```diff
@@ -0,0 +1,27 @@
+package pinball;
+
+public enum BodyType {
+    BOTTOM_WALL("BOTTOM_WALL"),
```

Comment:
What's the difference between all these wall types?

Reply:
Makes a difference when drawing them

Reply:
It doesn't.